lexicons/com/atproto/sync/getRepoStatus.json

```diff
@@ -0,0 +1,42 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.sync.getRepoStatus",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Get the hosting status for a repository, on this server. Expected to be implemented by PDS and Relay.",
+      "parameters": {
+        "type": "params",
+        "required": ["did"],
+        "properties": {
+          "did": {
+            "type": "string",
+            "format": "did",
+            "description": "The DID of the repo."
+          }
+        }
+      },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["did", "active"],
+          "properties": {
+            "did": { "type": "string", "format": "did" },
+            "active": { "type": "boolean" },
+            "status": {
+              "type": "string",
+              "description": "If active=false, this optional field indicates a possible reason for why the account is not active. If active=false and no status is supplied, then the host makes no claim for why the repository is no longer being hosted.",
+              "knownValues": ["takendown", "suspended", "deactivated"]
+            },
+            "rev": {
+              "type": "string",
+              "description": "Optional field, the current rev of the repo, if active=true"
+            }
+          }
```

Comment:
Should results in `sync.listRepos` also include the hosting status, and perhaps generally have the same shape as this?